src/components/page/blocks/QuestionBlock.jsx

```diff
@@ -31,50 +34,43 @@ class QuestionBlock extends React.PureComponent {
       updateAnswer,
     } = blockShared;
 
-    return (
-      <div className="qas">
-        {questions.map(question => {
-          const { question: q, number, tables } = question;
-          const primeQ = q[0];
-          const { id, questionType, qaReview, showUserAnswer } = primeQ;
-          const key = `qa-${id}`;
-
-          if (qaReviewPage && !qaReview) return false;
+    if (qaReviewPage && !qaReview) return false;
```

Comment:
the major change is in `QuestionBlock`, previously all questions were passed to `QuestionBlock` and then mapped into individual questions, now individual questions are passed to the block and displayed.